components/eventing-controller/pkg/backend/nats/jetstream/jetstream_test.go

```diff
@@ -1476,3 +1477,103 @@ func TestSubscriptionSubjectIdentifierNamespacedName(t *testing.T) {
 		})
 	}
 }
+
+// TestJetStream_NoNATSSubscription tests if the error is being triggered
+// when expected in-memory representation of NATS subscriptions are missing.
```

Comment:
instead of in-memory lets use the term `subscription map` or `internal subscription map`?